validator/src/main/resources/expected-data-template/dotnet/k8s/aws-sdk-call-trace.mustache

```diff
@@ -0,0 +1,64 @@
+[{
+  "name": "^{{serviceName}}$",
+  "http": {
+    "request": {
+      "url": "^{{endpoint}}/aws-sdk-call$",
+      "method": "^GET$"
+    },
+    "response": {
+      "status": "^500$"
+    }
```

Comment:
Don't need to verify status, not important for what we care about. Here and everywhere else in other trace validations

Reply:
Done